azure-templates/node_setup_steps.yml

```diff
@@ -0,0 +1,4 @@
+steps:
```

Comment:
we could make `azure-templates` folder hidden (e.g. rename it to `.azure-templates`)